snapcraft/tests/test_commands_gated.py

```diff
@@ -63,9 +63,10 @@ def test_gated_success(self):
         main([self.command_name, 'ubuntu-core'])
 
         expected_output = textwrap.dedent("""\
-            Name      Approved
-            snap-1           3
-            snap-2           5""")
+            Name      Revision  Required    Approved
+            snap-1           3  True        2016-09-19T21:07:27Z
```

Comment:
I think it's really the timezone information that's gross. Can we localize it? Or just show UTC?